apigee_edge.info.yml

```diff
@@ -12,8 +12,7 @@ dependencies:
   - drupal:filter
   - drupal:options
   - key:key
-  - drupal:system (>=8.7.0)
 
 configure: apigee_edge.settings
 
-php: "7.1"
+php: "8.0"
```

Comment:
Can you add 7.4 as the minimum php version we support.


Past June when Drupal 10.x is ready we will update this to 8.0